homework-submission/src/Components/updateTodo.css

```diff
@@ -0,0 +1,64 @@
+#enter-new-description {

```

Comment:
Is there a reason why are using ids here and not classes?
I always try to use classes and very rarely use ids, [you can find out why in this article](https://dev.to/claireparker/reasons-not-to-use-ids-in-css-4ni4)

Reply:
Yes I try to use classes too whenever I can, unless I know this element is special and its style won't be replicated
Was operating on autopilot I guess 😀, thank you so much Claudio, I'm working on my homework and following your feedback, which was very helpful 👐

Reply:
@keepitterron I kept using id instead of class because the styles weren't being applied for some reason